runtime/sema/authaccount_type.go

```diff
@@ -222,6 +243,45 @@ const authAccountTypePrivatePathsFieldDocString = `
 All the private paths of an account
 `
 
+const authAccountForEachPublicDocString = `
+Iterate over all the public paths of an account.
+
+Takes two arguments: the first is the path (/domain/key) of the stored object, and the second is the runtime type of that object.
```

Comment:
The iteration functions accept one argument, the iteration function. The iteration function then is passed two arguments.